test/unit/math/mix/mat/fun/log_sum_exp_test.cpp

```diff
@@ -1,431 +1,50 @@
-#include <stan/math/mix/mat.hpp>
-#include <gtest/gtest.h>
-#include <test/unit/math/rev/mat/fun/util.hpp>
+#include <test/unit/math/test_ad.hpp>
+#include <limits>
 #include <vector>
 
-using stan::math::fvar;
-using stan::math::log_sum_exp;
-using stan::math::var;
+TEST(MathMixMatFun, logSumExp) {
+  auto f = [](const auto& x) { return stan::math::log_sum_exp(x); };
 
-TEST(AgradMixMatrixLogSumExp, vector_fv_1st_deriv) {
-  using stan::math::vector_fv;
+  // TODO(carpenter): fix log_sum_exp boundary behavior
+  // Eigen::VectorXd x0(0);
+  // stan::test::expect_ad(f, x0);
```

Comment:
This one already has a separate issue so maybe just remove the comment. I will handle that issue once this is merged.

Reply:
removed comment.